.github/workflows/update_file.yaml

```diff
@@ -0,0 +1,38 @@
+name: Update files
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '43 1 * * 1'
+
+jobs:
+
+  pull-from-sunbeam:
+    name: Pull files from sunbeam
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout local repository
+        uses: actions/checkout@v4
+
+      - name: set git config
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          git config --global user.email "${GITHUB_ACTOR_ID}+${GITHUB_ACTOR}@users.noreply.github.com"
+          git config --global user.name "$(gh api /users/${GITHUB_ACTOR} | jq .name -r)"
+          git config -l
```

Comment:
I think we should use github application, because like this we will not have signed commit.

Reply:
I've tried https://github.com/actions/create-github-app-token, and it seems working fine.
Example: https://github.com/canonical/snap-tempest-automation/blob/main/.github/workflows/update-snapcraft.yaml#L77C13-L97